drupalcat/config/install/core.base_field_override.node.drupal_cat_project.promote.yml

```diff
@@ -0,0 +1,22 @@
+uuid: 1c5bd5f0-fcbc-4721-ad01-1aed89e32022
```

Comment:
Acabo de parlar amb Alex Pott (CMI) i m'ha confirmat que els UUIDs no s'han d'esborrar.

Al comentar-li el problema que tenim amb drush make, em contesta "don't use drush make, use the Configuration Installer. That will solve your problem"